### ISSUE TITLE
chrome webdriver manager not working in jenkins

### ISSUE THREAD

**Issue by mdniyazhashmi** [SergeyPirogov/webdriver_manager#143]
Hi,

I am trying to execute the code using  chrome webdriver manager using selenium python in my local jenkins but I am getting this error but if I am executing code through command line it is working fine upon checking the registry entry for Google chrome it contains chrome version correctly.

![image](https://user-images.githubusercontent.com/67937819/90720515-4d892880-e2d4-11ea-9794-59d576532a7e.png)
 

10:53:08 browser_type = 'google-chrome'
10:53:08 
10:53:08     def chrome_version(browser_type=ChromeType.GOOGLE):
10:53:08         pattern = r'\d+\.\d+\.\d+'
10:53:08     
10:53:08         cmd_mapping = {
10:53:08             ChromeType.GOOGLE: {
10:53:08                 OSType.LINUX: 'google-chrome --version || google-chrome-stable --version',
10:53:08                 OSType.MAC: r'/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version',
10:53:08                 OSType.WIN: r'reg query "HKEY_CURRENT_USER\Software\Google\Chrome\BLBeacon" /v version'
10:53:08             },
10:53:08             ChromeType.CHROMIUM: {
10:53:08                 OSType.LINUX: 'chromium --version || chromium-browser --version',
10:53:08                 OSType.MAC: r'/Applications/Chromium.app/Contents/MacOS/Chromium --version',
10:53:08                 OSType.WIN: r'reg query "HKEY_CURRENT_USER\Software\Chromium\BLBeacon" /v version'
10:53:08             },
10:53:08             ChromeType.MSEDGE: {
10:53:08                 OSType.MAC: r'/Applications/Microsoft\ Edge.app/Contents/MacOS/Microsoft\ Edge --version',
10:53:08                 OSType.WIN: r'reg query "HKEY_CURRENT_USER\SOFTWARE\Microsoft\Edge\BLBeacon" /v version',
10:53:08             }
10:53:08         }
10:53:08     
10:53:08         cmd = cmd_mapping[browser_type][os_name()]
10:53:08         stdout = os.popen(cmd).read()
10:53:08         version = re.search(pattern, stdout)
10:53:08         if not version:
10:53:08 >           raise ValueError(f'Could not get version for Chrome with this command: {cmd}')
10:53:08 E           ValueError: Could not get version for Chrome with this command: reg query "HKEY_CURRENT_USER\Software\Google\Chrome\BLBeacon" /v version
10:53:08 

Can you please look at this issue and let me know if I am missing anything.

Regards,
Niyaz

**Comment by SergeyPirogov** [comment]
Please try to execute both commands in your cmd:

```
 reg query "HKLM\SOFTWARE\Wow6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Google Chrome"
```

```
reg query "HKEY_CURRENT_USER\Software\Google\Chrome\BLBeacon" /v version
```

Let me know if they work

**Comment by mdniyazhashmi** [comment]
please find the screenshot for both queries, working as expected

![image](https://user-images.githubusercontent.com/67937819/90723499-78767b00-e2da-11ea-8ddd-107c4f3c4607.png)

**Comment by SergeyPirogov** [comment]
Please try v3.2.2. All tests are green

https://github.com/SergeyPirogov/webdriver_manager/runs/1006482578?check_suite_focus=true

**Comment by lavarith** [comment]
One reason that this occurred for me was that, because I had installed Jenkins as a service, the default "Log on" user for the service was a local account which didn't have access to the registry.

I clicked the windows icon, typed "Services.msc" and hit enter.
Then, I scrolled down to Jenkins, right clicked and hit properties. Then went to Log On and changed the log on user to the administrator on the account. Then I restarted the service.
